include/z64player.h

```diff
@@ -966,11 +1051,14 @@ typedef struct Player {
     /* 0xAE7 */ s8 unk_AE7; // a timer
     /* 0xAE8 */ s16 unk_AE8; // multipurpose timer
     /* 0xAEC */ f32 unk_AEC;
-    /* 0xAF0 */ Vec3f unk_AF0[2];
+    /* 0xAF0 */ union { // TODO: this may be an union of two structs
+                    Vec3f unk_AF0[2];
+                    f32 arr_AF0[6];
+                };
```

Comment:
Think this relates to the ocarina-playing stuff, doesn't it? The Player branch still has this as a Vec3f, I'm wondering where the array idea came from.

Reply:
This is one of those multi-purpose variables that's certainly used by the ocarina-playing stuff, but not limited to and includes non-related things as well.

For example, it stores info in the 5 deku pipes into `unk_AF0[0].y,z`/`unk_AF0[1].x,y,z` which is more of an array:
https://github.com/Decompollaborate/mm/blob/d2879b990f434a4f0980604e91fb495b2526ef18/src/overlays/actors/ovl_player_actor/z_player.c#L16193

But then you have instances like this where it's definitely being copied into as a `Vec3f`:
https://github.com/Decompollaborate/mm/blob/d2879b990f434a4f0980604e91fb495b2526ef18/src/overlays/actors/ovl_player_actor/z_player.c#L11412